contents/blog/open-source-stack-for-engineers.md

```diff
@@ -0,0 +1,160 @@
+- Third-party apps to enrich and move data
+
+## 2. [Metabase](/https://www.metabase.com/)
+
+![Metabase - open source data visualization for engineers](../images/blog/open-source-engineers/metabase-open-source.png)
+
+- **Useful for:** Data visualization, business intelligence<br/>
+- **Alternative to:** Looker, Tableau, PowerBI<br/>
+- **License:** [AGPL](https://en.wikipedia.org/wiki/Affero_General_Public_License)<br/>
+
+For visualizing data in a different way and running complex business intelligence, Metabase is an easy recommendation because it’s something we make use of at PostHog too, filling in the gap for competitors such as Looker.
+
+What makes Metabase so essential is that, like PostHog, it enables you to accomplish a huge amount without needing to resort to SQL. Instead, you can create BI dashboards in just a few minutes without ever needing to write a line of code. Afterwards, these dashboards can be shared anywhere — internally, or externally. 
+
+### Strengths
+- Easy to use, no SQL required
+- Interactive, drag and drop dashboards
+- Deploy to the cloud, or on-prem
+- Integrate with 20+ data sources
+
+## 3. [RudderStack](https://www.rudderstack.com/)
+
+![Rudderstack - open source data pipeline for engineers](../images/blog/open-source-engineers/rudderstack-open-source.png)
+
+- **Useful for:** CDP, Data pipeline<br/>
+- **Alternative to:** Segment, Tealium <br/>
+- **License:** [AGPLv3](https://en.wikipedia.org/wiki/Affero_General_Public_License)<br/>
+
+RudderStack has two things in common with PostHog. Firstly, it can act as a data pipeline to funnel ingested events to other sources, such as a data warehouse or third-party platform. Secondly, you need _two_ capital letters to spell it correctly. 
+
+Unlike PostHog however, RudderStack is completely focused on acting as a data pipeline. It can ingest data from more than 20 different sources, perform transformations in real-time and even perform version control operations via GitHub actions. 
+
+RudderStack is perfect for teams which are concerned with regulatory compliance, as it can mask PII and filter out sensitive events with ease. This has helped it become wildly popular with users such as Stripe, Hinge and Allbirds to name a few. 
+
+### Strengths
+- Transport data to anywhere
+- Transform data in real-time
+- Pre-defined schemas for data warehouses
+- SDKs track anonymous users and update downstream tools
+
+## 4. [NocoDB](https://www.nocodb.com/)
+
+![NocoDB - open source database collaboration for engineers](../images/blog/open-source-engineers/nocodb-open-source.png)
+
+- **Useful for:** Database collaboration<br/>
+- **Alternative to:** Airtable, Postgres <br/>
+- **License:** [AGPLv3](https://en.wikipedia.org/wiki/Affero_General_Public_License)<br/>
+
+NocoDB isn’t just an alternative to database tools like Airtable — it can also act as a tool which sits on top of platforms like Airtable, converting complex databases into ‘smart spreadsheets’. 
```

Comment:
```suggestion
[NocoDB](https://www.nocodb.com/) isn’t just an alternative to database tools like Airtable — it can also act as a tool which sits on top of platforms like Airtable, converting complex databases into ‘smart spreadsheets’. 
```